### PR TITLE
Add rendering of man_made=wastewater_plant, water_works

### LINKED ISSUE #717: Add rendering for man_made=wastewater_plant
The tag man_made=wastewater_plant should be rendered as icon and/or landuse.

See also https://trac.openstreetmap.org/ticket/1830.

### PR DESCRIPTION
**Pull request by Adamant36**
Fixes #717 
This pull request adds rendering of man_made=wastewater_plant and man_made=water_works in the industrial landuse color. Its my first pull request of this type. So, I might of added extra things that didn't need to be there or something. If so, feel free to let me know and I will correct it. I'm also not sure if the zoom levels are correct or not. I'll fix those also if need be. 

wastewater plant 
https://www.openstreetmap.org/#map=17/37.94864/-122.49576 (way) 
wastewater_plant way
https://www.openstreetmap.org/#map=19/37.89582/-122.52669 (node)
wastewater_plant node
water works 
https://www.openstreetmap.org/#map=19/38.60624/-121.52483 (way)
water_works way
water works node
water_works node

### CODE REVIEW

**Comment by kocio-pl** [comment]
I like this idea.

**Comment by Adamant36** [comment]
Me to. At least it didn't already have a PR for it :)

**Comment by kocio-pl** [comment]
There's a conflict to resolve caused by merging #3327.

**Comment by Adamant36** [comment]
Thanks for pointing it out. I resolved it.

**Comment by HolgerJeromin** [comment]
This was dicussed in the past, as landuse=industrial should be tagged in many cases...

File: amenity-points.mss [hunk 1]
```diff
@@ -40,12 +40,12 @@
   [feature = 'tourism_alpine_hut'][zoom >= 13],
   [feature = 'tourism_wilderness_hut'][zoom >= 13],
   [feature = 'amenity_shelter'][zoom >= 16] {
-    marker-file: url('symbols/shelter.svg');
```
Comment (HolgerJeromin) [inline amenity-points.mss]: Are these file moves from you or a merge / git artefact?
This should be done in a seperate PR.

File: amenity-points.mss [hunk 2]
```diff
@@ -40,12 +40,12 @@
...
+    marker-file: url('symbols/amenity/shelter.svg');
```
Comment (HolgerJeromin) [inline amenity-points.mss]: Are these file moves from you or a merge / git artefact?
This should be done in a seperate PR.

**Comment by Adamant36** [comment]
@HolgerJeromin, thanks for noticing it. it was from a different, earlier PR. I don't know why it did that, as they are on different branches and I don't remember having to commit all that stuff again when I did this one. Its showing up here as a commit though and its in the wastewater branch to when it shouldn't be. So I don't know. the whole thing is rather irritating. How do I go about cleaning it up properly?

**Comment by Adamant36** [comment]
@HolgerJeromin, I think I fixed it. Feel free to let me know if there is anything else wrong with it. Also, although it was discussed in the past there wasn't a consensus on it and still 5 years later only 10% of them are tagged with landuse=industrial. So whatever was or wasn't decided here, its clearly not what is happening with actual tagging. So I think they are worth rendering on their own. Instead of relying on people to tag them as landuse=industrial when its clearly not going to happen.

**Comment by kocio-pl** [comment]
Please wrap some long lines, but that's all - my test shows that it works OK. You have also made valid point about landuse tagging.

There are a lot of such objects and it's good that they will start to be visible at last:

taghistory 44

**Comment by Adamant36** [comment]
@kocio-pl, I wrapped some lines. So it should be good to go now.

**Comment by kocio-pl** [comment]
Thanks! Only about 10-15% of such objects have landuse tagged after ~10 years, so this is nice (and realistic) extension of the map.

**Pull request merged**